olad/Makefile.mk

```diff
@@ -63,14 +43,7 @@ ola_server_additional_libs += common/http/libolahttp.la
 endif
 
 # lib olaserver
-lib_LTLIBRARIES += olad/libolaserverplugininterface.la olad/libolaserver.la
-
-olad_libolaserverplugininterface_la_SOURCES = \
-    $(ola_server_plugin_interface_sources)
-olad_libolaserverplugininterface_la_CXXFLAGS = $(COMMON_CXXFLAGS)
-olad_libolaserverplugininterface_la_LIBADD = common/libolacommon.la \
-                                             common/web/libolaweb.la \
-                                             ola/libola.la
+lib_LTLIBRARIES += olad/plugin_api/libolaserverplugininterface.la olad/libolaserver.la
```

Comment:
olad/plugin_api/libolaserverplugininterface.la should be done in olad/plugin_api/Makefile.am